### ISSUE TITLE
Random connection loss that breaks status panel.

### ISSUE THREAD

**Issue by neeshacark** [yogstation13/Yogstation#19140]
### Severity

Round Breaking

### Short Description

This is not a new issue, I'm not entirely certain when it started besides that it was either in April or more likely last month - May.  This is not a 'me' issue.  Numerous people have been dealing with this.  Prior to these past few weeks I have scarcely crashed since the dark times of 2018-2020.  When I did crash, it was generally the whole server crashing or that my internet had fully gone out.  This is not that, and this has gotten to a point where it is impacting my enjoyment of the game (and I'm sure others as well).

### Reproduction

1. exist
2. ???
3. ???

### Round ID

n/a

### BYOND Username

Cark

### Was this done on a locally hosted or non-Yogstation hosted server?

None

**Comment by neeshacark** [comment]
If it needs to be said I highly doubt this is exploiters, ddosers or any other kind of griefer causing this issue.  This happens literally every round whether its 1 in the morning or 1 in the afternoon.

**Comment by 00aquir** [comment]
yeah, for me it was similar: when you use the reconnect command, it won't load. Just says "Loading...". Exiting and launching game restores it.
also, can we get a "fix stat panel" button in the ooc tab? there were times when the tabs loaded, but didn't load everything
